src/gui/sdlmain.cpp

```diff
@@ -4525,6 +4525,17 @@ extern void DEBUG_ShutDown(Section * /*sec*/);
 void MIXER_CloseAudioDevice(void);
 
 void restart_program(std::vector<std::string> & parameters) {
+#ifdef WIN32
+	std::string command_line = {};
+	bool first = true;
+	for (std::string &arg : parameters) {
```

Comment:
How about `for (const auto &arg : parameters)`?

Reply:
The functional programmer in me is crying 😢 

```
command_line = parameters.drop(1).join(" ")
```

Reply:
Is that line supposed to be real? It would be nice but won't compile so I assume not.

Reply:
Changed to auto

Reply:
> Is that line supposed to be real? It would be nice but won't compile so I assume not.

It was a joke 😆 That's how you do it in nicer languages like Kotlin or Scala, etc.